src/System.Runtime.TimeZoneData/System.Runtime.TimeZoneData.csproj

```diff
@@ -30,8 +36,13 @@
     <ItemGroup>
       <FileWrites Include="@(VersionFile)" />
     </ItemGroup>
+    <PropertyGroup>
+      <LatestTimeZoneDataVersion>@(VersionLines)</LatestTimeZoneDataVersion>
+    </PropertyGroup>
+  </Target>
 
-    <Warning Text="TimeZoneDataVerson does not match https://data.iana.org/time-zones/tzdb/version" Condition="'@(VersionLines)' != '$(TimeZoneDataVersion)'" />
+  <Target Name="CheckVersion" DependsOnTargets="GetLatestVersion">
```

Comment:
Who invokes this target and should this be an Error instead of a Warning?

Reply:
I still don't see anyone depending on this target. Is this dead code?

Reply:
yeah, I've removed it now